cmd/root.go

```diff
@@ -30,8 +28,9 @@ func Execute() {
 }
 
 func init() {
-	logger := supportlog.New()
-	logger.SetLevel(logrus.TraceLevel)
+	log.DefaultLogger = log.New()
+	log.DefaultLogger.SetLevel(logrus.TraceLevel)
```

Comment:
We would like to have fewer logs (debug) in production, perhaps we can move this to an option when running the service, then we pass the `LogLevel`.

Reply:
[84725af](https://github.com/stellar/wallet-backend/pull/14/commits/84725affc193651ede99e41b49e041b70f9c3e90)